components/Article.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+	<div>
```

Comment:
For an article, can you suggest a more semantically appropriate container to use?